Tools/tupleReadTest.C

```diff
@@ -1,20 +1,19 @@
 #include "NTupleReader.h"
```

Comment:
This is meant to be a very simple test function where we can do local testing.  Lets not commit all these changes